web/source/osk/visualKeyboard.ts

```diff
@@ -58,13 +58,13 @@ namespace com.keyman.osk {
     touchCount: number;
     currentTarget: KeyElement;
 
-    // Popup key management
+    // Used by embedded-mode's globe key
     menuEvent: KeyElement; // Used by embedded-mode.
+
+    // Popup key management
     keytip: KeyTip;
-    subkeyPopup: browser.SubkeyPopup;
-    browserPendingLongpress: browser.PendingLongpress;
-    embeddedPendingLongpress: any; // a temp, intermediate property during subkey abstraction work
-    subkeyDelegator: any; // a temp, intermediate property during subkey abstraction work
```

Comment:
One of the big goals of this PR... and the cause of most of this class's changes - actually using the newly-introduced gesture abstractions.